src/components/pages/Books.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable array-callback-return */
```

Comment:
- [ ] As stated before, kindly remove them and fix them. You can run this `npx eslint . --fix` to fix linters after installing them locally.